x-pack/plugins/security_solution/cypress/integration/exceptions/rule_details_flow/edit_exception.spec.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  EXCEPTION_EDIT_FLYOUT_SAVE_BTN,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  EXCEPTION_ITEM_CONTAINER,
+  FIELD_INPUT,
+} from '../../../screens/exceptions';
+import {
+  createExceptionList,
+  createExceptionListItem,
+  deleteExceptionList,
+} from '../../../tasks/api_calls/exceptions';
+import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
+import {
+  addExceptionEntryFieldValueOfItemX,
+  addExceptionEntryFieldValueValue,
+} from '../../../tasks/exceptions';
+import { ALERTS_COUNT, NUMBER_OF_ALERTS } from '../../../screens/alerts';
+
+describe('Edit exception from rule details', () => {
+  const exceptionList = getExceptionList();
+  const NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS = '1 alert';
+
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('exceptions');
+    login();
+  });
+
+  after(() => {
+    esArchiverUnload('exceptions');
+  });
+
+  beforeEach(() => {
+    deleteAlertsAndRules();
+    deleteExceptionList(exceptionList.list_id, exceptionList.namespace_type);
+    // create rule with exceptions
+    createExceptionList(exceptionList, exceptionList.list_id).then((response) => {
+      createCustomRuleEnabled(
+        {
+          ...getNewRule(),
+          customQuery: 'agent.name:*',
+          dataSource: { index: ['exceptions*'], type: 'indexPatterns' },
+          exceptionLists: [
+            {
+              id: response.body.id,
+              list_id: exceptionList.list_id,
+              type: exceptionList.type,
+              namespace_type: exceptionList.namespace_type,
+            },
+          ],
+        },
+        '2',
+        '2s'
+      );
+      createExceptionListItem(exceptionList.list_id, {
+        list_id: exceptionList.list_id,
+        item_id: 'simple_list_item',
+        tags: [],
+        type: 'simple',
+        description: 'Test exception item',
+        name: 'Sample Exception List Item',
+        namespace_type: 'single',
+        entries: [
+          {
+            field: 'unique_value.test',
+            operator: 'included',
+            type: 'match_any',
+            value: ['bar'],
+          },
+        ],
+      });
+    });
+
+    visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+    goToRuleDetails();
+    waitForTheRuleToBeExecuted();
+    waitForAlertsToPopulate();
+    goToExceptionsTab();
+  });
+
+  afterEach(() => {
+    esArchiverUnload('exceptions_2');
+  });
+
+  it('Edits an exception item', () => {
+    // displays existing exception item
+    cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 1);
+
+    editException();
+
+    // check that the existing item's field is being populated
+    cy.get(EXCEPTION_ITEM_CONTAINER)
+      .eq(0)
+      .find(FIELD_INPUT)
+      .eq(0)
+      .should('have.text', 'unique_value.test');
+
+    // check that you can select a different field
+    addExceptionEntryFieldValueOfItemX('agent.name{downarrow}{enter}', 0, 0);
+    addExceptionEntryFieldValueValue('foo', 0);
```

Comment:
Same here!

Reply:
Thank you thank you for your feedback. I've opened a follow up PR addressing it [here](https://github.com/elastic/kibana/pull/140448).